docs/docsite/rst/dev_guide/module_lifecycle.rst

```diff
@@ -1,50 +1,118 @@
 .. _module_lifecycle:
 
-**********************************
-The lifecycle of an Ansible module
-**********************************
+********************************************
+The lifecycle of an Ansible module or plugin
+********************************************
 
-Modules in the main Ansible repo have a defined life cycle, from first introduction to final removal. The module life cycle is tied to the `Ansible release cycle <release_cycle>`.
-A module may move through these four states:
+Modules and plugins in the main Ansible repository have a defined life cycle, from first introduction to final removal. The module and plugin life cycle is tied to the `Ansible release cycle <release_cycle>`.
+A module or plugin may move through these four states:
 
-1. When a module is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
+1. When a module or plugin is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
 
-2. If a module matures, we will remove the 'preview' mark in the documentation. We support (though we cannot guarantee) backwards compatibility for these modules, which means their parameters should be maintained with stable meanings.
+2. If a module or plugin matures, we will remove the 'preview' mark in the documentation. We support (though we cannot guarantee) backwards compatibility for these modules and plugins, which means their parameters should be maintained with stable meanings.
```

Comment:
again, not sure this is happening anymore. The old 'Status' section of the module docs doesn't show up after Ansible 2.9. I don't see status in `ansible-doc` output either.

Reply:
I tried to only minimally modify the first section, since I think it needs a major overhaul and I'm not sure what it should say...